style.css

```diff
@@ -0,0 +1,56 @@
+.header {
```

Comment:
- Kindly, by default, some HTML tags have some margin and padding, which leads your background image to have mrgin:8, so please reset it with bellow  :-1:  code 
```
* {
margin: 0;
padding: 0;
}
```